src/momento-redis-adapter.ts

```diff
@@ -275,7 +280,8 @@ export class MomentoRedisAdapter
         rsp = await this.momentoClient.set(
           this.cacheName,
           key,
-          value.toString()
+          value.toString(),
+          {compress: this.useCompression}
```

Comment:
Do we want to always use compression if the client is configured with compression?

Reply:
Yes we do. Because we are wrapping a Redis client API, the developer has no way to toggling per-key compression. For the first cut to keep things simple, when they enable compression on the client we will always compress/decompress.